src/main.rs

```diff
@@ -187,9 +193,14 @@ fn main() {
 }
 
 #[cfg(feature = "debug_ctap")]
-fn print_packet_notice(notice_text: &str, timer: &Timer) {
-    let now = timer.get_current_clock().flex_unwrap();
-    let now_us = (Timestamp::<f64>::from_clock_value(now).ms() * 1000.0) as u64;
+fn print_packet_notice(notice_text: &str, clock: &CtapClock) {
+    use core::convert::TryFrom;
+    use embedded_time::duration::Microseconds;
```

Comment:
Please move imports to the top of the file.

Reply:
Done